test/examples/full-form.json

```diff
@@ -74,17 +74,24 @@
       "label": "Security",
       "fields": [
         {
-          "label": "Security",
-          "type": "section_radio",
+          "label": "",
+          "type": "dropDown",
+          "required": false,
           "jsonPath": "",
+          "options": [
+            { "value": "", "label": "Select here" },
```

Comment:
See commend above. Let's start with Default properties being picked.

Reply:
ok will do the changes